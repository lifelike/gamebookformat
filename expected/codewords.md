# Gamebook #


Turn to 1 to begin.
<a name="section1">

## 1 ##
 Demonstrating how Codewords (AKA sightings) can be used. Go to [2](#section2)
. 
<a name="section2">

## 2 ##
 Got codeword **warrior**
. Simple enough to set a codeword. Turn to [9](#section9)
. 
<a name="section3">

## 3 ##
 That was easy.    
<a name="section4">

## 4 ##
 OK, if you have the codeword *warrior*
, you may turn to [3](#section3)
 otherwise you may go back to [2](#section2)
. Although we both know you have that codeword. If you have the codeword *fun*
 you may turn to [5](#section5)
, without it you can go to [8](#section8)
. 
<a name="section5">

## 5 ##
 Cheater! There is no way you can have codeword fun. 
<a name="section6">

## 6 ##
 If you have the codeword *fun*
 turn to [5](#section5)
. Otherwise you can go to the end at [3](#section3)
 or to the xortest at [4](#section4)
. 
<a name="section7">

## 7 ##
 If you have codeword *warrior*
 turn to [6](#section6)
. Otherwise you can go to the end at [3](#section3)
 or back to [1](#section1)
. 
<a name="section8">

## 8 ##
 This is just to demonstrate choices allowed when not having a codeword. Now go on to [7](#section7)
 (autotest) or [4](#section4)
 (xor test). 
<a name="section9">

## 9 ##
 If you have the codeword *warrior*
 you may turn to [3](#section3)
. If you do not have the codeword *fun*
, you may turn to [8](#section8)
. If you have the codeword *fun*
, you may instead turn to [5](#section5)
. Otherwise see [4](#section4)
. 

