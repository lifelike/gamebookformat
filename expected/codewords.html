<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'codewords';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 Demonstrating how <span class="collect" data-type="code"
    data-name="Codewords">Codewords</span> (AKA sightings) can be used. Go to <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section2"
     >
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 Got codeword <span class="add" data-type="code"
      data-what="warrior">warrior</span>. Simple enough to set a codeword. Turn to <a class="sectionref enabledlink" data-ref="9"
href="#section9">9</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>
<div class="section" id="section3"
     >
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 That was easy.    
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script>
<div class="section" id="section4"
     >
<div class="sectionnumber" id="para4">4</div>
<div class="sectiontext">
 OK, if you have the codeword <span class="has" data-type="code" data-what="warrior">warrior</span>, you may turn to <span class="xor"></span><a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> otherwise you may go back to <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. Although we both know you have that codeword. If you have the codeword <span class="has" data-type="code" data-what="fun">fun</span> you may turn to <span class="xor"></span><a class="sectionref enabledlink" data-ref="5"
href="#section5">5</a>, without it you can go to <a class="sectionref enabledlink" data-ref="8"
href="#section8">8</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(4, document.getElementById('section4'));
}
</script>
<div class="section" id="section5"
     >
<div class="sectionnumber" id="para5">5</div>
<div class="sectiontext">
 Cheater! There is no way you can have codeword fun. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(5, document.getElementById('section5'));
}
</script>
<div class="section" id="section6"
     >
<div class="sectionnumber" id="para6">6</div>
<div class="sectiontext">
 If you have the codeword <span class="has" data-type="code" data-what="fun">fun</span> turn to <span class="auto"></span><a class="sectionref enabledlink" data-ref="5"
href="#section5">5</a>. Otherwise you can go to the end at <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> or to the xortest at <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(6, document.getElementById('section6'));
}
</script>
<div class="section" id="section7"
     >
<div class="sectionnumber" id="para7">7</div>
<div class="sectiontext">
 If you have codeword <span class="has" data-type="code" data-what="warrior">warrior</span> turn to <span class="auto"></span><a class="sectionref enabledlink" data-ref="6"
href="#section6">6</a>. Otherwise you can go to the end at <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> or back to <a class="sectionref enabledlink" data-ref="1"
href="#section1">1</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(7, document.getElementById('section7'));
}
</script>
<div class="section" id="section8"
     >
<div class="sectionnumber" id="para8">8</div>
<div class="sectiontext">
 This is just to demonstrate choices allowed when not having a codeword. Now go on to <a class="sectionref enabledlink" data-ref="7"
href="#section7">7</a> (autotest) or <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a> (xor test). 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(8, document.getElementById('section8'));
}
</script>
<div class="section" id="section9"
     >
<div class="sectionnumber" id="para9">9</div>
<div class="sectiontext">
 If you have the codeword <span class="has" data-type="code" data-what="warrior">warrior</span> you may turn to <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a>. If you do not have the codeword <span class="hasnot" data-type="code"
      data-what="fun">fun</span>, you may turn to <a class="sectionref enabledlink" data-ref="8"
href="#section8">8</a>. If you have the codeword <span class="has" data-type="code" data-what="fun">fun</span>, you may instead turn to <a class="sectionref enabledlink" data-ref="5"
href="#section5">5</a>. Otherwise see <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(9, document.getElementById('section9'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
